frontend/hub/administration/repositories/RepositoryPage/RepositoryDetails.tsx

```diff
@@ -38,16 +38,19 @@ export function RepositoryDetails() {
   if (distroError || remoteError)
     return <HubError error={distroError || remoteError} handleRefresh={refresh} />;
   if (!distroData || !remoteData) return <LoadingPage breadcrumbs tabs />;
-  const distribution: { name: string; client_url: string } = distroData?.results[0];
+
+  const distribution: { name: string; client_url: string } | undefined = distroData?.results[0]
```

Comment:
`maybeUndefined ? maybeUndefined : undefined` .. this doesn't do anything

```suggestion
  const distribution: { name: string; client_url: string } | undefined = distroData?.results[0];
```